src/server/alembic/versions/05e0693f8cbb_key_value_table.py

```diff
@@ -0,0 +1,31 @@
+"""key/value table
+
+Revision ID: 05e0693f8cbb
+Revises: 6b8cf99be000
+Create Date: 2021-03-18 11:35:43.512082
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '05e0693f8cbb'
+down_revision = '6b8cf99be000'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
```

Comment:
was thinking we can have this a little more informative:
add columns:
- timestamp
- is_running (boolean)
- is_success (boolean)
- message (json or long string as you were doing for valcol)

Reply:
All possible, but scope creep beyond what that was intended to do. 

We should figure out our plan first.